yt/yt/server/query_tracker/yql_engine.cpp

```diff
@@ -40,13 +41,16 @@ class TYqlSettings
 {
 public:
     std::optional<TString> Stage;
+    EExecuteMode ExecuteMode;
 
     REGISTER_YSON_STRUCT(TYqlSettings);
 
     static void Register(TRegistrar registrar)
     {
         registrar.Parameter("stage", &TThis::Stage)
             .Optional();
+        registrar.Parameter("execution_mode", &TThis::ExecuteMode)
```

Comment:
So is it execution_mode or execute_mode?

Reply:
execution_mode in the API
ExecuteMode in variables in code